### PR TITLE
Add Buyee

### PR DESCRIPTION
**Pull request by contextnerror**

### CODE REVIEW

File: entries/b/buyee.jp.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Buyee": {
+    "domain": "buyee.jp",
+    "contact": {
+      "facebook": "buyee.jp.en",
+      "twitter": "buyee",
```
Comment (Carlgo11) [inline entries/b/buyee.jp.json]:
```suggestion
      "twitter": "Buyee",
```

**Review by Carlgo11 — DISMISSED** [review]
Is the site only available in Japan? If so, please specify that using a _regions_ element.

**Comment by contextnerror** [comment]
Nope, [several different regions.](https://buyee.jp/help/yahoo/guide/countries) They buy stuff from Japan and send it to you, hence the .jp domain.

**Comment by Carlgo11** [comment]
I'll let someone else review this site as I'm unable to view the site

File: entries/b/buyee.jp.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Buyee": {
+    "domain": "buyee.jp",
+    "contact": {
+      "facebook": "buyee.jp.en",
+      "twitter": "Buyee",
+      "email": "support@buyee.jp"
+    },
+    "keywords": [
+      "other"
```
Comment (phallobst) [inline entries/b/buyee.jp.json]: Even though they don't sell the goods themselves, the `retail` category is a better fit, I guess.
```suggestion
      "retail"
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**